src/process/tests.rs

```diff
@@ -141,26 +166,25 @@ async fn no_description_none_required_pr_actions() {
 #[tokio::test]
 async fn review_not_required_if_label_not_configured() {
     use ReviewState::{Approved, ChangesRequested, Commented};
-    macro_rules! assert_approved {
-        ($approved:expr, $cases:expr) => {{
-            let (pr, client, mut config) = make_context();
-            config.reviewed_label = None;
-            let mut analyzer = make_analyzer(&pr, &client, &config);
-            analyzer.reviews = RemoteData::Local($cases);
-            assert_eq!(
-                analyzer.required_actions().await.unwrap(),
-                *Actions::noop()
-                    .set_merge($approved)
-                    .set_label("ci-passed", Presence::Present)
-                    .set_label("needs-description", Presence::Absent)
-            );
-        }};
+
+    async fn assert_approved(approved: bool, cases: Vec<Review>) {
```

Comment:
This should make the backtrace still point to assertion

```suggestion
    #[track_caller]
    async fn assert_approved(approved: bool, cases: Vec<Review>) {
```